### ISSUE TITLE
[PL][Rogue][Spell][Pick Pocket]

### ISSUE THREAD

**Issue by Hextall** [Atlantiss/BugTracker#2385]
Umiejętność Pick Pocket w zastosowana na NPC z contentu Cataclysm pozwala na dzień dzisiejszy na uzyskanie wyłącznie pieniążków, co jest oczywiście bugiem. Przeleciałem szybko do Stranglethorn i okazało się za NPC 25-30 poprawnie lootują itemki, więc możliwe, że starszy content pod tym względem działa poprawnie. Nie jest to bug najwyższej wagi ale uniemożliwia na przykład farmienie Flame-Scarred Junkbox, które może się okazać niejednokrotnie dochodowym interesem dla klasy rogue.

Pick Pocket - http://www.wowhead.com/spell=921/pick-pocket
Flame-Scarred Junkbox - http://www.wowhead.com/item=63349/flame-scarred-junkbox

**Comment by Navedonar** [comment]
http://www.wowhead.com/item=63349/flame-scarred-junkbox dropi normalnie z mobów w stonecore co raczej nie powinno mieć miejsca

**Comment by Hextall** [comment]
Zgadza się. NPC, z których u nas lecą skrzynki nie mają żadnego specyficznego lootu. http://www.wowhead.com/npc=42428/devout-follower
